lib/components/exports.ts

```diff
@@ -1,4 +1,3 @@
 export * from "./Actions/exports"
 export * from "./Charts/exports"
-export * from "./Information/exports"
```

Comment:
Oh we need that, I don't think we should remove it

Reply:
If I remove the `Badge` component, there's nothing inside of the folder, right? Should I left it even if it's empty?